source/docs/magento2/i18n.md

```diff
@@ -0,0 +1,61 @@
+---
+id: magento2-i18n
+title: Magento2 i18n and l10n
+---
+
+Front-Commerce is build with internationalization (<abbr title="internationalization">i18n</abbr>) in mind.
+
+You can translate every message from your Front-Commerce application as detailed in the [Translate your application](/docs/advanced/theme/translations.html) page.
+If you are running a multi-language store, you can [configure a store per language](/docs/advanced/production-ready/multistore.html).
```

Comment:
Would it make sense to prepend this sentence by "In magento" or something like this? This sentence feels a bit odd to me otherwise.